code/model/modelreplace.cpp

```diff
@@ -189,8 +196,37 @@ void virtual_pof_init() {
 
 // Internal helper functions
 
+template<typename T, typename member_t>
+T object_copy_including_array_member(const T& item, member_t T::* ptm, int T::* size) {
+	T result{ item };
+	result.*ptm = (member_t) vm_malloc(sizeof(typename std::remove_pointer<member_t>::type) * (result.*size));
```

Comment:
Not in this case, since it's for cloning some structures within polymodel, and the respective structures in question have their fields allocated with malloc. More importantly, they're later deallocated with free, so if we insert stuff copied here, we must malloc

Reply:
Ok